packages/apm/src/transaction.ts

```diff
@@ -80,16 +84,9 @@ export class Transaction extends SpanClass {
   }
 
   /**
-   * Sets the finish timestamp on the current span.
-   *
-   * @inheritdoc
-   *
-   * @param trimEnd If true, sets the end timestamp of the transaction to the highest timestamp of child spans, trimming
-   * the duration of the transaction. This is useful to discard extra time in the transaction that is not
-   * accounted for in child spans, like what happens in the idle transaction Tracing integration, where we finish the
-   * transaction after a given "idle time" and we don't want this "idle time" to be part of the transaction.
+   * @inheritDoc
    */
-  public finish(endTimestamp?: number, trimEnd: boolean = false): string | undefined {
+  public finish(endTimestamp?: number): string | undefined {
```

Comment:
I was more thinking of `span.finish({ timestamp: 123 })` or `span.finish({ trimEnd: true })`, buuut this can work fine as well I guess.